src/redis.go

```diff
@@ -60,6 +61,8 @@ func main() {
 		os.Exit(0)
 	}
 
+	args.Hostname = hostnameIpv6Formater(args.Hostname)
```

Comment:
Rather than a custom function, we could use https://pkg.go.dev/net#JoinHostPort, which already handles this case.

Reply:
Thanks @roobre! This approach is better. There is new update in pull request to accomplish it.